sdk/nodejs/automation/tag.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * TagMap is a key-value map of tag metadata associated with a stack.
+ */
+export type TagMap = { [key: string]: string };
```

Comment:
Not strictly necessary. Workspace/LocalWorksapce/Stack methods could just return `{[key:string]: string}`. Included for client readability.